generator/AWSPSGeneratorLib/HelpMaterials/Examples/LM/Get-LMVersionsByFunction.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>Get-LMVersionsByFunction -FunctionName "MylambdaFunction123" -Select Versions</code>
```

Comment:
`-Select Versions` is the default for this cmdlet (see `man Get-LMVersionsByFunction -parameter select`), so it can be omitted. Could you also include the output of the cmdlet?